pkg/apis/backup/v1/backup_spec.go

```diff
@@ -26,6 +26,8 @@ type ArangoBackupSpec struct {
 	// Deployment
 	Deployment ArangoBackupSpecDeployment `json:"deployment,omitempty"`
 
+	EncryptionSecret *string `json:"encryptionSecret,omitempty"`
```

Comment:
EncryptionSecret  --> EncryptionSecretName ?

I am not sure how to rotate a secret, however if it's allowed to remove the secret after the rotate is done, this name is useless.
Shouldn't it contain a hash of the used key (when a backup is created). And this name as part of the restore only?

Reply:
This is related only to Backup - Key from this secret will be restored on all servers just for Restore Operation and removed after. But I can move it to Restore request in ArangoDeployment - good idea

To rotate EncryptionKey in ArangoDeployment - just change this secret or change secret name in ArangoDeployment.